wallet/src/account/constants.rs

```diff
@@ -13,7 +13,7 @@ pub(crate) const PARALLEL_REQUESTS_AMOUNT: usize = 500;
 
 /// ms before an account actually syncs with the network, before it just returns the previous syncing result
 /// this is done to prevent unnecessary simultaneous synchronizations
-pub(crate) const MIN_SYNC_INTERVAL: u128 = 200;
+pub(crate) const MIN_SYNC_INTERVAL: u128 = 5;
```

Comment:
Should we have a value just for testing or do you think it's also fine?

Reply:
I'm not sure, what this should prevent is that if you call .sync() and then .sync() again while the first one hasn't finished, that it doesn't immediately start syncing after it just finished, but just return directly
This depends on the sync mutex and that shouldn't take that long to be available again I guess
A higher value would also help if you call .sync() and .sync() right after it finished, but at the end if it's called twice (the second time after the first finished), then I think it should be okay to sync again